litex/soc/software/libbase/vsnprintf.c

```diff
@@ -262,6 +262,7 @@ int vsnprintf(char *buf, size_t size, const char *fmt, va_list args)
 
 			case 'X':
 				flags |= PRINTF_LARGE;
+                                /* FALLTHRU */
```

Comment:
Are you sure you wanted this to be part of the PR?

Reply:
See previous response...avoids the alternative of someone switching off the warning from the command line, which would defeat its purpose.

Reply:
I don't like these decorators, they look untidy in the code, but the warnings are worse.